packages/dropdown-react/src/Dropdown.tsx

```diff
@@ -1,71 +1,115 @@
-import React from "react";
-import Downshift from "downshift";
+// @ts-ignore
+import CoreToggle from "@nrk/core-toggle/jsx";
+import React, { useEffect, useState } from "react";
+import { v4 as uuid } from "uuid";
+import { useListNavigation } from "./useListNavigation";
 
 interface Props {
     label: string;
     items: string[];
     className?: string;
-    onChange: (selection: string) => void;
     initialInputValue?: string;
+    onChange?: (value: string) => void;
 }
 
-export const Dropdown = ({ label, items, className = "", initialInputValue, onChange }: Props) => {
-    const onSelectionChange = (selection: string) => onChange(selection);
+interface CoreToggleSelectEvent {
+    detail: { textContent: string };
+    target: { hidden: boolean; button: HTMLButtonElement; value: { textContent: string } };
+}
+
+function lower(value: string | undefined) {
```

Comment:
Du you need to call toString(), with typeguard and nul check first?

Or you could 
```
const lower = (str="") => str.toLowerCase().replace(/\s/g,"")
```

Reply:
The component accepts numbers as values, so they have to be converted to strings, unfortunately. Your suggestion is a bit more elegant, though!